packages/playground/src/main.ts

```diff
@@ -9,9 +9,19 @@ import vuetify from "@/plugins/vuetify";
 import App from "./App.vue";
 import { defineGlobals } from "./config";
 import router from "./router";
+import { normalizeError } from "./utils/helpers";
 
 const app = createApp(App);
 
+app.config.errorHandler = error => {
+  console.error(
+    "[Uncaught Error]\n" +
+      `- Message: ${normalizeError(error, "Something went wrong in dashboard app but we couldn't fetch it.")}\n` +
+      `- Stack: ${error && typeof error === "object" ? Reflect.get(error, "stack") : null}\n` +
+      `- Constructor: ${error && typeof error === "object" ? error.constructor.name : null}`,
+  );
+};
+
```

Comment:
Why do we do this here?

Reply:
To show the most possible available data from error object